### DISCUSSION TITLE
How do I make my start page ("home") a blog?

### DISCUSSION THREAD

**Discussion by joho1968** [typemill/typemill#483] [General]
I'm sure I've missed it and that it's in the documentation somewhere, but how do I make the Typemill site a blog from the start?

I can't seem to edit those attributes in the meta data for "home" ... :thinking: 

![image](https://github.com/typemill/typemill/assets/3135227/cfa1beba-427e-4150-856d-cbfa370a8204)

**Comment by trendschau** [comment]
Yes, sorry, that is not super intuitive, (blogging is not a primary feature of typemill). 

If you want to show posts on the homepage, then you have to configure this in the theme (if the theme has that option), since the startpage is technically not a folder and does not contain anything. See screenshot of the cyanine theme settings.

![Unbenannt](https://github.com/typemill/typemill/assets/2714547/59266851-c23d-448d-b681-2986e24d3521)
